src/dnn_detect.cpp

```diff
@@ -72,14 +81,45 @@ class DnnNode {
     cv::dnn::Net net;
     cv::Mat resized_image;
 
+    volatile bool triggered;
+    volatile bool processed;
+
+    dnn_detect::DetectedObjectArray results;
+
+    ros::ServiceServer detect_srv;
+
+    bool detectCallback(dnn_detect::Detect::Request &req,
```

Comment:
Can we name this something like `triggerCallback` detect is kinda confusing, since it isn't a callback for a detection, but rather to start one.

Reply:
Yes, good point